packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart

```diff
@@ -20,7 +20,7 @@ class XcodeBuildSystemMigration extends ProjectMigrator {
   @override
   void migrate() {
     final File? xcodeWorkspaceSharedSettings = _xcodeWorkspaceSharedSettings;
-    if (xcodeWorkspaceSharedSettings == null || !xcodeWorkspaceSharedSettings.existsSync()) {
+    if (xcodeWorkspaceSharedSettings == null) {
```

Comment:
Still need this one.

Reply:
Ooh you're right, added it back in https://github.com/flutter/flutter/pull/114099/commits/224a5c40654121f40026004b915ca8a554527ae8.